### ISSUE TITLE
Getting an optimized model with TOCO failed

### ISSUE THREAD

**Issue by abiela** [tensorflow/models#6819]
### System information
- **What is the top-level directory of the model you are using**: tensorflow
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Windows 10
- **TensorFlow installed from (source or binary)**: source
- **TensorFlow version (use command below)**: 1.12.0 (CPU only)
- **Bazel version (if compiling from source)**: 0.19.0
- **CUDA/cuDNN version**: 9.0/7.4
- **GPU model and memory**: GeForce 660M/8 GB RAM
- **Exact command to reproduce**: 

```
bazel run -c opt tensorflow/contrib/lite/toco:toco -- --input_file=D:\tensor\output\tflite_graph.pb --output_file=D:\tensor\output\flutter_model.tflite --inference_type=QUANTIZED_UINT8 --input_format=TENSORFLOW_GRAPHDEF --output_format=TFLITE --input_shapes=1,300,300,3 --input_arrays=normalized_input_image_tensor --output_arrays='TFLite_Detection_PostProcess','TFLite_Detection_PostProcess:1','TFLite_Detection_PostProcess:2','TFLite_Detection_PostProcess:3' --mean_values=128 --std_values=128 --change_concat_input_ranges=false --allow_custom_ops
```

### Describe the problem
My goal is to run a Mobilenet pretrained model on a mobile. I went step by step through the following tutorial, starting from building TF from source:
https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/running_on_mobile_tensorflowlite.md

Everything went quite smoothly or I managed to find some already solved issues. Yet, I have a problem with the last step that is getting the optimized model with TOCO.

### Source code / logs

When I ran the command:

```
bazel run -c opt tensorflow/contrib/lite/toco:toco -- --input_file=D:\tensor\output\tflite_graph.pb --output_file=D:\tensor\output\flutter_model.tflite --inference_type=QUANTIZED_UINT8 --input_format=TENSORFLOW_GRAPHDEF --output_format=TFLITE --input_shapes=1,300,300,3 --input_arrays=normalized_input_image_tensor --output_arrays='TFLite_Detection_PostProcess','TFLite_Detection_PostProcess:1','TFLite_Detection_PostProcess:2','TFLite_Detection_PostProcess:3' --mean_values=128 --std_values=128 --change_concat_input_ranges=false --allow_custom_ops
```

I got the following error:
```
2019-05-18 13:50:34.792006: I tensorflow/contrib/lite/toco/import_tensorflow.cc:1080] Converting unsupported operation: TFLite_Detection_PostProcess
2019-05-18 13:50:34.870333: F tensorflow/contrib/lite/toco/tooling_util.cc:854] Check failed: model.HasArray(output_array) Output array not found: 'TFLite_Detection_PostProcess'
```

**Comment by Ezaldeen99** [comment]
I don't know if this the correct solution but I solved it removing '' from the  output_arrays 

I used this command  
`bazel run -c opt tensorflow/lite/toco:toco -- --input_file=D:/programming/tensorflow-master_2/tensorflow-master/tmp/tflite_graph.pb --output_file=D:/programming/tensorflow-master_2/tensorflow-master/tmp/detect.tflite --input_format=TENSORFLOW_GRAPHDEF --output_format=TFLITE --input_shapes="1,300, 300,3" --input_arrays=normalized_input_image_tensor --output_arrays=TFLite_Detection_PostProcess,TFLite_Detection_PostProcess:1,TFLite_Detection_PostProcess:2,TFLite_Detection_PostProcess:3  --inference_type=QUANTIZED_UINT8 --mean_values=128 --std_values=128.0 --change_concat_input_ranges=false --allow_custom_ops --default_ranges_min=0 --default_ranges_max=6`

**Comment by jvishnuvardhan** [comment]
@abiela  Please let us know whether @Ezaldeen99 suggestion worked for you. Thanks!

**Comment by jvishnuvardhan** [comment]
Automatically closing this out since I understand it to be resolved, but please let me know if I'm mistaken.Thanks!
